packages/common/container-definitions/api-report/container-definitions.api.md

```diff
@@ -13,11 +13,12 @@ import type { IClientDetails } from '@fluidframework/protocol-definitions';
 import type { IDisposable } from '@fluidframework/core-interfaces';
 import type { IDocumentMessage } from '@fluidframework/protocol-definitions';
 import type { IDocumentStorageService } from '@fluidframework/driver-definitions/internal';
-import { IErrorBase } from '@fluidframework/core-interfaces';
+import { IErrorBase } from '@fluidframework/core-interfaces/internal';
+import type { IErrorBase as IErrorBase_2 } from '@fluidframework/core-interfaces';
```

Comment:
Note: This is expected, given the way api-extractor works, but noting in case we have trouble with IErrorBase later.

Reply:
We eliminated a lot of these before by correcting some funny things. Why is this one expected?
My guess would be that it comes into being when there are mismatched /internal versus not /internal cases.

Anyway, I think people need to be aware of when these are okay and when they are not. I wish there were annotations on why.

Reply:
My mistake.  We can fix it by re-exporting from the public .d.ts.  (#20420.)